.github/workflows/deploy-env.yaml

```diff
@@ -0,0 +1,48 @@
+name: Deploy Environment
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Environment to deploy to'
+        type: choice
+        required: true
+        default: 'ops-preview-1'
+        options:
+        options:
+        - ops-preview-1
+      region:
+        description: 'Region containing cluster to deploy to'
+        type: choice
+        required: true
+        default: 'us-east-1'
+        options:
+        - us-east-1
+        - us-east-2
+        - eu-central-1
```

Comment:
We should have multiple listed environments such as `ops-preview-1@us-east-1` so people would not need to guess the environments, also some of the environments have `v2` postfix

Reply:
✅